cmd/zc_traverser_blob.go

```diff
@@ -129,6 +129,7 @@ func (t *blobTraverser) traverse(preprocessor objectMorpher, processor objectPro
 			blobPropertiesResponseAdapter{blobProperties},
 			common.FromAzBlobMetadataToCommonMetadata(blobProperties.NewMetadata()), // .NewMetadata() seems odd to call, but it does actually retrieve the metadata from the blob properties.
 			blobUrlParts.ContainerName,
+			"", // versionID field is an attribute of blobs only.
```

Comment:
>versionID field is an attribute of blobs only. [](start = 10, length = 46)

minor: fix comment since this is the blob traverser

Reply:
Done